websocket/core/src/main/java/io/undertow/websockets/handshake/Base64.java

```diff
@@ -12,15 +12,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.undertow.websockets.jsr.handshake;
+package io.undertow.websockets.handshake;
+
+import io.undertow.websockets.JsrWebSocketLogger;
 
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-import io.undertow.UndertowLogger;
-
 /**
  * <p>
  * Encodes and decodes to and from Base64 notation.
```

Comment:
Any reason to not use the Base64 encoders/decoders provided by the JDK?

Reply:
Because this code was written to work on JDK7 for EAP6 :-)